src/gui_common/CustomRichTextLabel.cs

```diff
@@ -262,25 +262,54 @@ private static string ParseCustomTagsString(string extendedBbcode)
     /// <param name="input">The string enclosed by the custom tags</param>
     /// <param name="bbcode">Custom Thrive bbcode-styled tags</param>
     /// <param name="attributes">Attributes specifying additional functionalities to the bbcode.</param>
-    private static string BuildTemplateForTag(string input, ThriveBbCode bbcode, List<string> attributes)
+    private string BuildTemplateForTag(string input, ThriveBbCode bbcode, List<string> attributes)
```

Comment:
What was the change that means this can no longer be static?

Reply:
I've come to the conclusion that this being static is not necessary, it's private thus won't be used in any other places, secondly in the future it might benefit knowing its current rich text label instance to do operation/accessing information regarding them for use in some additional tags.

Reply:
This being static makes the code easier to reason about. This being static said "I won't touch the calling object" so you could just call that method and not have to think if it changed the current state of the object. This no longer being static increases the mental load on future programmers (and me who is trying to review this code).

Reply:
Alright then, static it is.